src/guides/v2.3/graphql/payment-methods/braintree-vault.md

```diff
@@ -20,7 +20,7 @@ The following diagram shows the workflow for placing an order when Braintree Vau
 
 1. The client renders the token information, and the customer selects a payment method.
 
-   When the customer selects a stored payment method, the PWA uses the [`setPaymentMethodOnCart`]({{page.baseurl}}/graphql/mutations/set-payment-method.html) mutation to set the payment method to `braintree_cc_vault`. The vaulted public hash is passed with other optional properties in the [`braintree_cc_vault`](#braintree_cc_vault-object).
+   When the customer selects a stored payment method, the PWA uses the [`setPaymentMethodOnCart`]({{page.baseurl}}/graphql/mutations/set-payment-method.html) mutation to set the payment method to [`braintree_cc_vault`](#braintree_cc_vault-object). The vaulted public hash is passed with other optional properties in the [`braintree_cc_vault`](#braintree_cc_vault-object).
```

Comment:
Is this second link really necessary as it also appears elsewhere in the sentence? 

I'd say move the link from the second to the first instance.

Reply:
It's not necessary but we can keep it for smooth user experience.